test/systemtests/network_test.go

```diff
@@ -401,6 +411,9 @@ func (s *systemtestSuite) TestNetworkAddDeleteTenantFwdModeChangeVXLAN(c *C) {
 				PvtSubnet:        "172.19.0.0/16",
 			}), IsNil)
 			time.Sleep(60 * time.Second)
+			if s.basicInfo.Scheduler == kubeScheduler {
```

Comment:
same as above.

Also, here the 60s sleep is outside of setupinfranetwork call. In other files (hostaccess_test.go) it is inside the if block.